src/scss/06_components/paragraphs/_list-of-links.scss

```diff
@@ -150,10 +151,12 @@ $-item-spacing: $spacing-triple;
 
   .list-of-links__item__title {
     @include font('h4');
-    margin: 0 0 0 (100px + $spacing);
+    margin-block: 0;
+    margin-inline: (100px + $spacing) 0;
 
     @include breakpoint($breakpoint-s) {
       margin: 0;
+      margin-inline-start: 0;
```

Comment:
Tested and looks like it doesn't so the margin-inline-start is needed.
Might be because the `margin-inline-start: (100px + $spacing);` (from earlier lines) seems to be under more specific selector `[dir="rtl"] .list-of-links__item .list-of-links__item__title` and the `margin: 0` goes under `.list-of-links__item .list-of-links__item__title`.

But the `margin: 0` is actually not needed so I removed it.